generators/entity-client/templates/react/src/main/webapp/app/entities/entity.tsx.ejs

```diff
@@ -60,29 +61,7 @@ import { APP_DATE_FORMAT, APP_LOCAL_DATE_FORMAT } from 'app/config/constants';
 import { ITEMS_PER_PAGE } from 'app/shared/util/pagination.constants';
 <%_ } _%>
 
-export interface I<%= entityReactName %>Props {
-  getEntities: ICrudGetAllAction<I<%= entityReactName %>>;
-  <%_ if (pagination === 'infinite-scroll') { _%>
-  reset: Function;
-  <%_ } _%>
-  <%_ if (searchEngine === 'elasticsearch') { _%>
-  getSearchEntities: ICrudSearchAction<I<%= entityReactName %>>;
-  <%_ } _%>
-  <%= entityInstance %>List: I<%= entityReactName %>[];
-  <%_ if (pagination === 'infinite-scroll') { _%>
-  links: {
-    last: 0
-  };
-  <%_ } _%>
-  <%_ if (pagination !== 'no') { _%>
-  totalItems: 0;
-  location: any;
-  <%_ } _%>
-  <%_ if (pagination === 'pagination' || pagination === 'pager') { _%>
-  history: any;
-  <%_ } _%>
-  match: any;
-}
+export interface I<%= entityReactName %>Props extends StateProps, DispatchProps, RouteComponentProps<{url: string}> {}
```

Comment:
If it works well for autocompletion and refactoring in IDE/editors then this will be great